README.md

```diff
@@ -1,94 +1,83 @@
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.github.premnirmal/TextCounter/badge.png)](http://search.maven.org/#artifactdetails|com.github.premnirmal|TextCounter|1.1.0|)
-[![Android Arsenal](https://img.shields.io/badge/Android%20Arsenal-TextCounter-brightgreen.svg?style=flat)](https://android-arsenal.com/details/1/1140)
-
 # TextCounter
 
-A simple subclass of android's [TextView](http://developer.android.com/reference/android/widget/TextView.html) that counts up or down.
-Customize the counting using the attributes below.
+A HMOS library which counts up/down of the text value.
 
-See the demo project for more implementation details.
+## Source
+Inspired by [premnirmal/TextCounter](https://github.com/premnirmal/TextCounter) - version 1.1.0
 
-![](img/counter.gif)
+## Feature
+A simple subclass of harmony's Text that counts up or down. Customize the counting using the attributes below.
 
-## Download
+See the demo project for more implementation details.
 
-### Gradle
+<img src="https://github.com/priyankabb153/TextCounter/blob/main/screenshots/TextCounterDemo.gif" width="256">
```

Comment:
Make sure this local url is pointing to the gif present in this project's "screenshots" folder

Reply:
Done